src/main/kotlin/no/nav/syfo/client/DokArkivClient.kt

```diff
@@ -96,7 +99,7 @@ class DokArkivClient(
             val oidcToken = oidcClient.oidcToken()
             header("Authorization", "Bearer ${oidcToken.access_token}")
             header("Nav-Callid", msgId)
-            body = FerdigstillJournal("9999")
+            body = FerdigstillJournal("9999") // TODO: journalfoerendeEnhet må fylles ut når vi har den
```

Comment:
Er det virkelig bare enheten vi må ha med når vi ferdigstiller? Må vi ikke ha med ident til saksbehandler også..?

Reply:
Vi har bare sendt 9999 før, men hvorvidt det er riktig vet jeg ikke enda...

Reply:
Sorry, sjekket swaggeren for dette apiet (https://dokarkiv-q1.nais.preprod.local/swagger-ui.html#/arkiver-og-journalfoer-rest-controller/ferdigstillJournalpostUsingPATCH), og det er riktig at vi kun sender enheten her :) 

Men ved ferdigstillelse av oppgave må vi ha begge deler, se https://oppgave.nais.preprod.local/#/Oppgave/patchOppgave Du kan sende veileders ident nå og 9999 som enhet, så tar vi enheten som en egen oppgave sånn som vi hadde planlagt uansett :)

Reply:
Det er tilordnetRessurs som skal settes til veileders ident forresten :)

Reply:
Tusen takk, @tiafiring !